test/GitcoinGovernor.t.sol

```diff
@@ -923,19 +945,25 @@ contract NewGitcoinGovernorProposalTest is
     _jumpToActiveProposal(_newProposalId);
 
     // Delegates vote with a mix of For/Against/Abstain with For winning.
-    vm.prank(PROPOSER); // kbw.eth (~1.8M votes)
+    vm.prank(delegates[0].addr);
     governorBravo.castVote(_newProposalId, FOR);
-    vm.prank(0x2df9a188fBE231B0DC36D14AcEb65dEFbB049479); // janineleger.eth (~1.53M)
+    vm.prank(delegates[1].addr);
     governorBravo.castVote(_newProposalId, FOR);
-    vm.prank(0x4Be88f63f919324210ea3A2cCAD4ff0734425F91); // kevinolsen.eth (~1.35M)
+    vm.prank(delegates[2].addr);
     governorBravo.castVote(_newProposalId, FOR);
-    vm.prank(0x34aA3F359A9D614239015126635CE7732c18fDF3); // Austin Griffith (~1.05M)
+    vm.prank(delegates[3].addr);
     governorBravo.castVote(_newProposalId, AGAINST);
-    vm.prank(0x7E052Ef7B4bB7E5A45F331128AFadB1E589deaF1); // Kris Is (~1.05M)
+    vm.prank(delegates[4].addr);
     governorBravo.castVote(_newProposalId, ABSTAIN);
-    vm.prank(0x5e349eca2dc61aBCd9dD99Ce94d04136151a09Ee); // Linda Xie (~1.02M)
+    vm.prank(delegates[5].addr);
     governorBravo.castVote(_newProposalId, AGAINST);
 
+    // The vote should pass.
+    assertGt(
```

Comment:
Can you help me understand what the point of these assertions is? Is it to make sure that the the delegates selected actually will produce the vote intended by the tests? I.e. so that if we updated the fork block in the future and the weights no longer work out this assertion will fail so we know we need to adjust the tests?

If that's right--or whatever the real reason is--do you mind adding a brief comment to make that clear?

Reply:
> so that if we updated the fork block in the future and the weights no longer work out this assertion will fail so we know we need to adjust the tests?

Yup, that's exactly right. Sure, I'll add a comment

Reply:
Done in 5140d80